src/db/oracledb/books-dao.js

```diff
@@ -0,0 +1,130 @@
+import _ from 'lodash';
+
+import oracledb from 'oracledb';
+
+import { parseQuery } from 'utils/parse-query';
+
+import { getConnection } from './connection';
+
+
+
+/**
+ * Return a list of books
+ *
+ * @param {object} query Query parameters
+ * @returns {Promise<object>[]} Promise object represents a list of books
+ */
+const getBooks = async (query) => {
+  const connection = await getConnection();
+  try {
+    const parsedQuery = parseQuery(query); // contrib is changed
+    const { rawBooks } = await connection.execute(parsedQuery);
```

Comment:
Wait what? What is `parsedQuery`? Is it a SQL query that can be run?

Reply:
This is already implemented in Pets skeleton API, I believe it is a general GET by query, so still have to dive into this one when I move to this endpoint. For now, this method is just copied from Pets GET.